main/tray.js

```diff
@@ -29,6 +47,13 @@ function init () {
   windows.main.on('hide', updateTrayMenu)
 }
 
+function checkLinuxTraySupport (cb) {
+  cp.exec('dpkg --get-selections libappindicator1', function (err, stdout) {
```

Comment:
dpkg is debian (and derivates) specific, not linux.This won't in other distros (red hat, fedora, etc). If you want to ensure libappindicator1 is installed you need to add it to dependencies on package creation